### PR TITLE
DAOS-2060 tools: Add daos cont get-acl command

### PR DESCRIPTION
**Pull request by kjacque**
- Add daos cont get-acl command. This includes the input
  parameters --outfile, --force, and --verbose.
- Add client API endpoint for getting cont ACL.
- Add daos_ace_str_get_verbose() to expand an ACE
  for verbose mode.
- Fixed a bug where an ACE was allowed to have no
  access types.
- Fixed a bug where the ACL wouldn't be returned
  over IV if it was empty.

Signed-off-by: Kris Jacque <kristin.jacque@intel.com>

### CODE REVIEW

File: src/client/api/container.c [hunk 1]
```diff
@@ -156,6 +156,30 @@ daos_cont_query(daos_handle_t coh, daos_cont_info_t *info,
...
+	if (prop == NULL)
+		return -DER_NOMEM;
+
+	prop->dpp_entries[0].dpe_type = DAOS_PROP_CO_ACL;
+	prop->dpp_entries[1].dpe_type = DAOS_PROP_CO_OWNER;
+	prop->dpp_entries[2].dpe_type = DAOS_PROP_CO_OWNER_GROUP;
+
+	*acl_prop = prop;
+
+	return daos_cont_query(coh, NULL, prop, ev);
```
Comment (mchaarawi) [inline src/client/api/container.c]: shouldn't you free prop if query fails?

Comment (kjacque) [reply src/client/api/container.c]: Good catch, will fix.

Comment (bfaccini) [reply src/client/api/container.c]: Well, space will be freed soon upon exit due to error. I still ask myself each time if this cleanup code really needs to be present... Only for the client/command case for sure ;-)  At least, it is required to make Valgrind happy ??

**Comment by kjacque** [comment]
The Functional test failure is a known issue, [DAOS-3945](https://jira.hpdd.intel.com/browse/DAOS-3945).

**Pull request merged**